### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by SergiyVS**
https://SergiyVS.github.io/layout_catalog/
https://SergiyVS.github.io/layout_catalog/report/html_report/

### CODE REVIEW

File: src/styles/blocks/header.scss [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  @extend %main-normal-text;
+  font-weight: $font-weight-primary;
+  font-size: $font-size-main;
+  line-height: 14px;
+  text-transform: uppercase;
+  color: $color-main;
+
+  @extend %display-flex-spcBtw;
+  // display: flex;
+  // justify-content: space-between;
```
Comment (qoa11a) [inline src/styles/blocks/header.scss:19]: Don't forget to remove comments.

File: src/styles/blocks/card.scss [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+.card {
+  width: 200px;
+
+  background: $color-background-main;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  @extend %display-flex-cntr;
+  // display: flex;
+  // justify-content: center;
```
Comment (qoa11a) [inline src/styles/blocks/card.scss:13]: Remove comments here and elsewhere.

**Review by qoa11a — APPROVED** [review]
Good job!

**Review by Anastasiia-Svintsova — APPROVED** [review]

**Comment by SergiyVS** [comment]
Don't show in MA results page

**Review by Anastasiia-Svintsova — APPROVED** [review]

**Review by ihor-jpeg — APPROVED** [review]

**Review by polosanya — APPROVED** [review]

**Pull request closed without merging**